### ISSUE TITLE
[REQUEST] New Libreoffice icons

### ISSUE THREAD

**Issue by thelonewarrior** [B00merang-Artwork/Windows-10#12]
Can you please update to use new office icons?

**Comment by Elbullazul** [comment]
You mean appm icons, like these?

![office-2019](https://cdn.dribbble.com/users/55180/screenshots/5677309/icons.png)

**Comment by GitUser445** [comment]
Yes, I believe that's what he means. I would also like that.

**Comment by Elbullazul** [comment]
yes, I think I can add them in a couple of weeks, depending on how other stuff goes

**Comment by Elbullazul** [comment]
also referenced [here](https://github.com/B00merang-Project/Windows-10/issues/190)
